### PR TITLE
docs: modify 021.md, 021 CN version and README

### PR DESCRIPTION
**Pull request by JoannaWu84**

### CODE REVIEW

File: newsletters/021.md [hunk 1]
```diff
@@ -2,10 +2,14 @@
 
 ## Editor's Note
 
-Welcome to the twenty-first issue of the SG2042 Newsletter. 
+Welcome to the twenty-first issue of the SG2042 Newsletter. Thank you for your continued interest and trust and hope you will enjoy this update.
 
 ## Highlights
 
++ SOPHGO, in collaboration with Chinasoft International, has established a training and certification centre for IT application innovation talents and a mobile internet training room for Beijing Polytechnic College. This partnership aims to jointly cultivate more internationally-oriented, versatile technical talents and promote the sustainable development of the information technology industry.
```
Comment (XieJiSS) [inline newsletters/021.md:9]: 或许 mobile interconnected training room？mobile internet 我第一眼以为是手机互联网（虽然说 i 大写才是互联网

File: newsletters/021.md [hunk 2]
```diff
@@ -14,7 +18,7 @@ Most of the code is already open-source and can be obtained from repositories su
 
 + Sophgo Community work: https://github.com/sophgo/linux-riscv
 
-  + 
+  + Device tree configuration
```
Comment (XieJiSS) [inline newsletters/021.md:21]: Updated device tree configuration

File: newsletters/021.md [hunk 3]
```diff
@@ -25,23 +29,43 @@ Most of the code is already open-source and can be obtained from repositories su
...
 
 We're looking for fun, good, or profitable use cases for SG2042. Feel free to share your experiences with us - [just send a PR!](https://github.com/sophgocommunity/SG2042-Newsletter/pulls)
 
 ## Events and Games
 
++ Although there are no specific activities planned this week, we encourage you to explore our previous editions and engage with us through our social media channels.
 
 ## In the News
 
++ [How many ways are there to drive the AHT10 on the Milk-V Duo?][news-1]
```
Comment (XieJiSS) [inline newsletters/021.md:50]: an AHT10
a Milk-V Duo

File: newsletters/021.md [hunk 4]
```diff
@@ -25,23 +29,43 @@ Most of the code is already open-source and can be obtained from repositories su
...
+
++ [Troubleshooting for Milk-V Duo Camera][news-2]
```
Comment (XieJiSS) [inline newsletters/021.md:52]: for --> a

Troubleshoot 可以做及物动词

**Review by XieJiSS — CHANGES_REQUESTED** [review]

**Comment by JoannaWu84** [comment]
已修改

**Pull request merged**